Source/MicrowaveNetworks/Touchstone/TouchstoneOptions.cs

```diff
@@ -13,9 +13,19 @@ public class TouchstoneOptions
         public ParameterType Parameter = ParameterType.Scattering;
         /// <summary>Specifies the format of the network paramater data pairs in the file.</summary>
         public FormatType Format = FormatType.MagnitudeAngle;
-        /// <summary>Specifies the reference resistance in ohms, where <see cref="Resistance"/> is a real, positive number of ohms.</summary>
+        /// <summary>
+        /// Specifies the reference resistance in ohms, where <see cref="Resistance"/> is a real, positive number of ohms.
+        /// If the <see cref="TouchstoneParameterAttribute"/> "R" is complex it will be represented by its real part. 
+        /// </summary>
         [TouchstoneParameter("R")]
         public float Resistance = 50;
+        /// <summary>
+        /// Specifies the reference reactance in ohms, where <see cref="Reactance"/> is a real number of ohms. 
+        /// If the <see cref="TouchstoneParameterAttribute"/> "R" is complex it will be represented by its imaginary part.
+        /// Otherwise it is considered to be 0.
+        /// </summary>
+        [TouchstoneParameter("R")]
+        public float Reactance = 0;
```

Comment:
Might be worth a comment here that this isn't directly a part of the Touchstone standard but added to support tools like scikit